test/framework/cluster_proxy.go

```diff
@@ -206,6 +213,32 @@ func (p *clusterProxy) GetClientSet() *kubernetes.Clientset {
 	return cs
 }
 
+func (p *clusterProxy) GetPodInformer(ctx context.Context) cache.Informer {
+	if p.cache == nil {
```

Comment:
sync.Once is probably nice here 
(cf with other occurences in the CAPI codebase)